eng/jacoco-test-coverage/pom.xml

```diff
@@ -275,7 +275,7 @@
     <dependency>
       <groupId>com.azure.resourcemanager</groupId>
```

Comment:
What is the expected date for GA @weidongxu-microsoft ?

Reply:
GA packages is published to maven yesterday.
https://mvnrepository.com/artifact/com.azure.resourcemanager